doc/fwutil/fwutil.md

```diff
@@ -392,18 +392,50 @@ _sonic-buildimage/device/<platform_name>/<onie_platform>/platform_components.jso
         "Module1": {
             "component": {
                 "CPLD": {
-                    "firmware": "/etc/<platform_name>/fw/<onie_platform>/module1/cpld.bin",
+                    "firmware": "/usr/share/sonic/<platform_name>/<onie_platform>/fw_update/cpld.bin",
                     "version": "10"
                 },
                 "FPGA": {
-                    "firmware": "/etc/<platform_name>/fw/<onie_platform>/module1/fpga.bin",
+                    "firmware": "/usr/share/sonic/<platform_name>/<onie_platform>/fw_update/fpga.bin",
                     "version": "5"
                 }
             }
         }
     }
 }
 ```
+3. Platform component fw update configuration with platform specific script and boot options
+```json
+{
+    "chassis": {
+        "Chassis1": {
+            "component": {
+                "BIOS": {
+                    "firmware": "/usr/share/sonic/<platform_name>/<onie_platform>/fw_update/bios.bin",
+                    "script": "/usr/share/sonic/<platform_name>/<onie_platform>/fw_update/bios_fw_update",
+                    "version": "0ACLH003_02.02.010"
+                    "required_reboot": "None"
+                    "immediate_action": "No"
```

Comment:
@sujinmkang IMHO, this should be fetched automatically by Platform API if needed.
The config file should be simple and clear.

**Use case:** the same firmware package may have a different behaviour based on device vendor/model.
**Example:** SSD with/without on-the-fly update mode